internal/tray-agent/agent/client/dashboard-client.go

```diff
@@ -0,0 +1,186 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"os"
+	"strings"
+)
+
+const (
```

Comment:
Are those default values or are they hardcoded values? Configuration should be taken from external environment/configuration files.

Reply:
Yes they are default values. Since the aim of the button action is to launch the Dashboard, I tried to reduce the processing time specifically switching for each resource from a two-step procedure (List w/ listOptions + find) to a single step (this does not apply obviously to pods). I assumed that enforcing a specific name just for the LiqoDash resources was an acceptable trade-off to gain a quicker response.

Do you think a List solution is more suitable?